packages/augur-sdk/src/state/getter/Users.ts

```
@@ -532,17 +536,20 @@ export class Users {
     const universe = params.universe
       ? params.universe
       : await augur.getMarket(params.marketId).getUniverse_();
-    const profitLossSummary = await Users.getProfitLossSummary(augur, db, {
+    profitLossSummary = await Users.getProfitLossSummary(augur, db, {
       universe,
       account: params.account,
     });
-
+  } catch(e) {
```

leaving try catch in b/c exceptions get swallowed up by sdk